packages/cashier/src/stores/payment-agent-transfer-store.ts

```diff
@@ -91,25 +124,32 @@ export default class PaymentAgentTransferStore {
         general_store.setLoading(true);
         this.onRemount = this.onMountPaymentAgentTransfer;
         await general_store.onMountCommon();
-        if (!this.transfer_limit.min_withdrawal) {
-            const response = await payment_agent.getPaymentAgentList();
+        if (!this.transfer_limit.min) {
+            const response: PaymentAgentListResponse = await payment_agent.getPaymentAgentList();
             const current_payment_agent = await this.getCurrentPaymentAgent(response);
             this.setMinMaxPaymentAgentTransfer(current_payment_agent);
         }
         general_store.setLoading(false);
     }
 
     @action.bound
-    requestTryPaymentAgentTransfer = async ({ amount, currency, description, transfer_to }) => {
+    requestTryPaymentAgentTransfer = async ({
+        amount,
+        currency,
+        description,
+        transfer_to,
+    }: PaymentAgentTransferRequest) => {
         this.error.setErrorMessage('');
+
         const payment_agent_transfer = await this.WS.authorized.paymentAgentTransfer({
             amount,
             currency,
             description,
             transfer_to,
             dry_run: 1,
         });
-        if (+payment_agent_transfer.paymentagent_transfer === 2) {
+
+        if (payment_agent_transfer.paymentagent_transfer === 2) {
```

Comment:
just a query are we receiving then in number format only

Reply:
@vinu-deriv Yes based on the types coming from `@deriv/api-types` these fields are `number` so we don’t need to case anymore. If we trust the types 😁